.taskcluster.yml

```diff
@@ -12,7 +12,7 @@ tasks:
           - master
     payload:
       maxRunTime: 3600
-      image: python
+      image: escapewindow/cot-gpg-keys
```

Comment:
Nit: We should move this docker image to https://hub.docker.com/u/mozillareleng/, in order to avoid permissions issues.

Reply:
+1. I was putting that in the future-wontfix pile if and when we move off gpg keys, but it might be worth doing anyway.